extension/src/popup/views/__tests__/Account.test.tsx

```diff
@@ -168,6 +256,85 @@ describe("Account view", () => {
       expect(screen.getAllByText("USDC")).toBeDefined();
     });
   });
+  it("displays balances and scam notifications on custom Mainnet network", async () => {
```

Comment:
these tests also give us a little cover on the custom network experience, which we sorely need

Reply:
nice, thanks